templates/service/it/HealthEndpointIntegrationSpec.scala

```diff
@@ -0,0 +1,40 @@
+package uk.gov.hmrc.$!APP_PACKAGE_NAME!$
+
+import org.scalatest.concurrent.{IntegrationPatience, ScalaFutures}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import org.scalatestplus.play.guice.GuiceOneServerPerSuite
+import play.api.Application
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.ws.WSClient
+
+class HealthEndpointIntegrationSpec
```

Comment:
Now we've added this, can we remove `MicroserviceIntegrationSpec`?
MicroserviceIntegrationSpec isn't really testing `AppConfig` (which could be tested without inflating the whole application) - it's there to serve the same purpose as `HealthEndpointIntegrationSpec` - i.e. does the app start as a whole.